reconcile/openshift_base.py

```diff
@@ -250,11 +254,12 @@ def wait_for_namespace_exists(oc, namespace):
         raise Exception(f'namespace {namespace} does not exist')
 
 
-def apply(dry_run, oc_map, cluster, namespace, resource_type, resource,
-          wait_for_namespace, recycle_pods=True):
+def apply(dry_run, oc_map: OC_Map, cluster, namespace, resource_type, resource,
+          wait_for_namespace, recycle_pods: bool = True,
+          privileged: bool = False):
```

Comment:
While you're at it, can you please complete the type signature here? At least the return type?